corehq/sql_db/util.py

```diff
@@ -62,6 +62,46 @@ def run_query_across_partitioned_databases(model_class, q_expression, values=Non
             yield result
 
 
+def paginate_query_across_partitioned_databases(model_class, q_expression, annotate=None, query_size=5000):
```

Comment:
`query_size` isn't passed in anywhere so might be nice to drop it for now. any reason it isn't set to DJANGO_MATCH_BATCH_SIZE by default?

Reply:
That comes from the warehouse, for bulk inserts and this is (theoretically) a generic sql accessor. They do not seem like they need to be connected beyond the fact that they are both used in this code. It seemed odd to tie the two functionalities together.